CMakeLists.txt

```diff
@@ -1093,7 +1093,6 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/util/logger.cpp
   src/util/logging.cpp
   src/util/mac.cpp
-  src/util/moc_included_test.cpp
```

Comment:
Why does this no longer work? Does it error with a false positive?

Reply:
It always fails on my machine. If you want to try yourself, just checkout this branch and remove the commit to see if it works for you.